### DISCUSSION TITLE
blocked account

### DISCUSSION THREAD

**Discussion by Didfishnet** [vercel/community#4185] [Account Issue] [answered]
### Question

when i click sign up with github it says account blocked

**Comment by Faizan-Iqbal-07** [comment]
Same problem with me

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by Faizan-Iqbal-07** [comment]
Still i'm facing login issue . please unblock my account
check this email : ***@***.***

**Comment by Faizan-Iqbal-07** [comment]
please unblock my account as urgent as possible
